### ISSUE TITLE
Mismatched Swift Versions

### ISSUE THREAD

**Issue by SDGGiesbrecht** [swiftwasm/carton#401]
The following are from the 0.19.0 docker tag. Note the mismatch between 5.8 and 5.7.3.

```
carton test
- checking Swift compiler path: /root/.carton/sdk/wasm-5.8.0-RELEASE/usr/bin/swift
Inferring basic settings...
- swift executable: /root/.carton/sdk/wasm-5.8.0-RELEASE/usr/bin/swift
SwiftWasm Swift version 5.7.3 (swiftlang-5.7.3)
Target: x86_64-unknown-linux-gnu
```

```
/root/.carton/sdk/wasm-5.8.0-RELEASE/usr/bin/swift build -c debug --product WorkspacePackageTests -Xswiftc -color-diagnostics --triple wasm32-unknown-wasi
Process failed and produced following output: 
Error: Process failed with non-zero exit status and following output:
 and following error output:
error: 'workspace': package 'workspace' is using Swift tools version 5.8.0 but the installed version is 5.7.3
```

Is there a way to work around it with extra flags or something to get Swift 5.8 working now, or do we have to wait for the next release?

**Comment by kateinoigakukun** [comment]
Oops! Thank you for letting me know so quickly! I've released a new patched release 0.19.1 to fix the issue. Could you try it again with the new version?

**Comment by SDGGiesbrecht** [comment]
Works great with 0.19.1. Thanks.
